server/athenian/api/align/models.py

```diff
@@ -16,3 +16,68 @@ def __init__(self, success: bool):
     def success(self) -> bool:
         """Whether the removal operation was successful or not."""
         return self._success
+
+
+class MutateGoalResultGoal(Model):
+    """The goal nested inside MutateGoalResult."""
+
+    attribute_types = {
+        "id": int,
+    }
+
+    def __init__(self, id: int):
+        """Init the MutateGoalResultGoal."""
+        self._id = id
+
+    @property
+    def id(self) -> int:
+        """Get the id of the goal."""
+        return self._id
+
+
+class MutateGoalResult(Model):
+    """The result of a GraphQL mutation about a goal."""
+
+    attribute_types = {
+        "goal": MutateGoalResultGoal,
+    }
+
+    def __init__(self, goal: MutateGoalResultGoal):
+        """Init the MutateGoalResultGoal."""
+        self._goal = goal
+
+    @property
+    def goal(self) -> MutateGoalResultGoal:
+        """Get the id of the goal."""
+        return self._goal
+
+
+class CreateGoalInputFields:
```

Comment:
```suggestion
from enum import Enum

class CreateGoalInputFields(Enum):
```

Reply:
`Enum` requires the bloat `.value` when using the field, is this ok ?

Reply:
Ok, then
```python
from athenian.api.models.web.base_model_ import Enum
class CreateGoalInputFields(metaclass=Enum):
```

Reply:
done